sql/descriptor.go

```diff
@@ -111,12 +113,22 @@ func (p *planner) createDescriptor(plainKey descriptorKey, descriptor descriptor
 	b.CPut(descKey, descriptor, nil)
 	// Mark transaction as operating on the system DB.
 	p.txn.SetSystemDBTrigger()
-	return p.txn.Run(&b)
+	if err := p.txn.Run(&b); err != nil {
+		return err
+	}
+	dcache[string(key)] = descriptor
```

Comment:
How about creating a descriptor cache that gets populated on a read instead of during creation. I also think that having a ttl of 5seconds on the read cache would be nice to keep it close to consistent.